dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -10,17 +10,31 @@ local tputests = base.BaseTest {
 
   timeout: 900, # 15 minutes, in seconds.
 
-  image: std.extVar('image'),
-  imageTag: std.extVar('image-tag'),
+  image: 'pytorchlightning/pytorch_lightning:base-xla-py3.6-torch{PYTORCH_VERSION}',
+  imageTag: '{PR_NUMBER}-{PYTORCH_VERSION}',
```

Comment:
seems like `image` and `imageTag` are not used

Reply:
about commit SHA, won't be there collision if the user sends another build before the first ends?
also include python version...

Reply:
I shouldn't need `imageTag` as my base image is directly pulled from DockerHub. https://hub.docker.com/r/pytorchlightning/pytorch_lightning/tags?page=1&ordering=last_updated&name=xla

Reply:
<img width="660" alt="Screenshot 2021-02-22 at 10 28 57" src="https://user-images.githubusercontent.com/12861981/108695979-d63d8700-74f8-11eb-87e4-2ca766cfc405.png">

Reply:
Yes, I assumed it would work as one block. But jsonnet expect ImageTag separated from image name.